nss-day-cohort-43-csharp/MVC-Green-Monkeys
# Lw branch3

# Description
Admin can now create a new category.

## Type of change

# Testing Instructions
Login as an admin, go to Category Management in the Nav bar.
Click "create category"
Write category name, click "create"
New category should populate in the list, search for it alphabetically. 

# Checklist:
